woocommerce/woocommerce
# Monorepo: Add JS formatter and linter

### Changes proposed in this Pull Request:

Add Prettier and ESlint configurations to monorepo root

### How to test the changes in this Pull Request:

1. Make sure you have Prettier and ESlint support in your IDE.
2. Visit any js page and see eslint errors.
3. If you have format on save, save the file and see Prettier format the file according to Gutenberg rules.
